Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -52,7 +53,7 @@ public function formatExamples()
         return [
             'DateTime object' => [$dateTime, 'Y-m-d H:i:s', '2013-07-03 12:34:56'],
             'timestamp as integer' => [1372856513, 'Y-m-d', '2013-07-03'],
-            'timestamp as string' => ['1372856513', 'Y-m-d', '2013-07-03'],
+            'timestamp as string' => ['@1372856513', 'Y-m-d', '2013-07-03'],
```

Comment:
Well, yeah, I'm basically fine with the change of behaviour per se, just not sure if we should add it in a minor release.